pages/docs/couple-your-code/couple-your-code-waveform.md

```diff
@@ -8,9 +8,6 @@ summary: "With waveform iteration, you can interpolate coupling data in time for
 {% experimental %}
 These API functions are work in progress, experimental, and are not yet released. The API might change during the ongoing development process. Use with care.
 {% endexperimental %}
-{% note %}
-This feature is only available for parallel implicit coupling. An extension to serial implicit coupling is planned. Without loss of generality, we moreover only discuss the API functions `readBlockVectorData` and `writeBlockVectorData` in the examples.
-{% endnote %}
```

Comment:
Somewhere we should still mention that this is only for implicit coupling.

Reply:
I modified the note.